src/libcrun/utils.c

```diff
@@ -1140,8 +1140,10 @@ copy_from_fd_to_fd (int src, int dst, int consume, libcrun_error_t *err)
       cleanup_free char *buffer = NULL;
       ssize_t remaining;
 
+#define BUFFER_SIZE 4096
+
 #ifdef HAVE_COPY_FILE_RANGE
-      nread = copy_file_range (src, NULL, dst, NULL, 0, 0);
+      nread = copy_file_range (src, NULL, dst, NULL, BUFFER_SIZE, 0);
```

Comment:
from the `copy_file_range` man page:

```
It copies up to len bytes of data from the source file descriptor fd_in  to  the  target  file  descriptor
fd_out,  overwriting  any  data that exists within the requested range of the
target file.
```

Could we just use `SIZE_MAX` here instead of `BUFFER_SIZE`?

Reply:
No reason not to.  I only used `BUFFER_SIZE` for parity with the `read()`/`write()` path.  I'll treat that as your preference and tweak it.

Reply:
Hmm, using `SIZE_MAX` triggers an `EOVERFLOW` error in one of the podman tests, and I suppose it would be surprising for the consume=0 case.  Going back to `BUFFER_SIZE`.